packages/core/content-type-builder/admin/src/components/ComponentCard/index.js

```diff
@@ -106,7 +106,7 @@ function ComponentCard({ component, dzName, index, isActive, isInDevelopmentMode
         paddingRight={4}
       >
         <StackCentered spacing={1}>
-          <StyledFontAwesomeIcon icon={icon} />
+          <StyledFontAwesomeIcon icon={icon || 'dice-d6'} />
```

Comment:
I think we should put this default value in a contants file so we don't need to update it everywhere. Wdyt?

Reply:
Indeed 🤔. We could add it to the helper-plugins but it might be a bit overkill 😅